standards/naming-domains.md

```diff
@@ -0,0 +1,55 @@
+---
+category: Operating services
+---
+# Naming domains
+
+All services with real users and which look like GOV.UK should be on
+`*.service.gov.uk` or `*.service.justice.gov.uk`.
+
+### service.gov.uk
+
+The [service standard](https://www.gov.uk/service-manual/technology/get-a-domain-name)
+already sets out when and how to get a `*.service.gov.uk` subdomain:
+public-facing services which have passed their beta assessment should
+be able to get one.
+
+### service.justice.gov.uk
+
+All other services with real users and which look like GOV.UK should be
+on here, including:
+
+- public-facing services which haven’t yet passed their beta assessment
+(but may be in private beta with a restricted set of real users)
+- internal services which look like GOV.UK (although [they can’t use the
+crown, New Transport font etc](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk#if-your-service-isnt-on-govuk))
+
+## Staff-facing sites
+
+Any staff-facing sites which exist as part of public-facing services
+should also be on the `*.service(.justice).gov.uk` domain.
+
+For example `staff.prisonvisits.service.gov.uk` is the staff-facing
+side of `www.prisonvisits.service.gov.uk`.
+
+## Pre-production environments
+
+This standard includes pre-production environments. For those, use this
+domain naming pattern:
+
+`<application>.<environment>.<service-name>.service(.justice).gov.uk`
+
+A real example would be `staff.staging.prisonvisits.service.gov.uk`.
```

Comment:
I've expanded this pattern and included an example of how to use it for a service which has multiple applications per environment. This is based on the assumption that environments are not shared across services, which may not be true everywhere.